core/rfb.js

```diff
@@ -624,8 +641,10 @@ export default class RFB extends EventTargetMixin {
         }
 
         const size = this._screenSize();
+        size.w = Math.floor(size.w / this._display.scale);
+        size.h = Math.floor(size.h / this._display.scale);
```

Comment:
I'm not sure we want to tie this in to the display scaling right now. I would prefer if you just use `window.devicePixelRatio` and `this._nativeDpi` here.